.github/workflows/rebuildDependencies.yml

```diff
@@ -229,12 +241,30 @@ jobs:
           --build=never \
           --no-remote \
           > "$graphFile"
+
         conan list \
           --graph "$graphFile" \
           --graph-context=build-only \
           --format=json \
-          > "$packageListFile"
-        conan remove --list "$packageListFile" --confirm
+          > $buildPackagesFile
+
+        conan list \
+          --graph "$graphFile" \
+          --graph-context=host \
+          --format=json \
+          > $hostPackagesFile
+
+        # Some are both build and host requirements and must not be removed
+        # => exclude them from the list
+        cat $buildPackagesFile $hostPackagesFile | jq -s '
```

Comment:
unclear why this is needed, no info in PR description

Reply:
I don't fully understand why build-context package is required on "client" side if everything has already been built. What's the role of protobuf, is protoc running at CMake configure or build time?